### PR TITLE
[WIP] PHPC-941: Add MongoDB\BSON\toExtendedJSON() to wrap bson_as_extended_json()

### PR DESCRIPTION
**Pull request by derickr**
https://jira.mongodb.org/browse/PHPC-941

### CODE REVIEW

File: tests/bson-corpus/code_w_scope-decodeError-011.phpt [hunk 1]
```diff
@@ -1,7 +1,5 @@
 --TEST--
 Javascript Code with Scope: bad scope doc (field has bad string length)
---XFAIL--
-Depends on PHPC-889
```
Comment (jmikola) [inline tests/bson-corpus/code_w_scope-decodeError-011.phpt]: Shouldn't this still fail until [PHPC-889](https://jira.mongodb.org/browse/PHPC-889) is resolved?

File: tests/bson-corpus/int64-valid-004.phpt [hunk 2]
```diff
@@ -1,7 +1,5 @@
 --TEST--
 Int64 type: 0
---XFAIL--
-Depends on PHPC-941 and PHP encoding integers as 32-bit if range allows
```
Comment (jmikola) [inline tests/bson-corpus/int64-valid-004.phpt]: I don't recall the original context, but this now passes with PHPC-941 implemented? If so, I suppose "PHP encoding integers as 32-bit if range allows" wasn't really relevant?

File: tests/bson-corpus/int64-valid-003.phpt [hunk 3]
```diff
@@ -1,7 +1,5 @@
 --TEST--
 Int64 type: -1
---XFAIL--
-Depends on PHPC-941 and PHP encoding integers as 32-bit if range allows
```
Comment (jmikola) [inline tests/bson-corpus/int64-valid-003.phpt]: Same comment regarding "PHP encoding integers as 32-bit if range allows".

File: tests/bson-corpus/code_w_scope-decodeError-011.phpt (hunk 1 above)
Comment (derickr) [inline tests/bson-corpus/code_w_scope-decodeError-011.phpt]: Probably, but your script stripped this line out.

File: tests/bson-corpus/int64-valid-003.phpt (hunk 3 above)
Comment (derickr) [inline tests/bson-corpus/int64-valid-003.phpt]: Probably, but your script stripped this line out.

File: tests/bson-corpus/int64-valid-004.phpt (hunk 2 above)
Comment (derickr) [inline tests/bson-corpus/int64-valid-004.phpt]: Probably, but your script stripped this line out.

**Comment by jmikola** [comment]
Superseded by https://github.com/mongodb/mongo-php-driver/pull/578.

**Pull request closed without merging**